WcaOnRails/app/views/persons/_personal_records.html.erb

```diff
@@ -18,30 +18,32 @@
       </thead>
       <tbody>
         <% @ranks_single.sort_by { |rank_single| rank_single.event.rank } .each do |rank_single| %>
-          <% rank_average = @ranks_average.find { |rank| rank.event_id == rank_single.event_id } %>
-          <tr>
-            <td class="event" data-event="<%= rank_single.event.id %>">
-              <%= cubing_icon rank_single.event.id %>
-              <%= t "events.#{rank_single.event.id}" %>
-            </td>
-            <%= rank_td rank_single, "country" %>
-            <%= rank_td rank_single, "continent" %>
-            <%= rank_td rank_single, "world" %>
-            <td class="single">
-              <%= link_to rankings_path(rank_single.event_id, "single"), class: "plain" do %>
-                <%= rank_single.solve_time.clock_format %>
-              <% end %>
-            </td>
-            <td class="average">
-              <%= link_to rankings_path(rank_single.event_id, "average"), class: "plain" do %>
-                <%= rank_average&.solve_time&.clock_format %>
-              <% end %>
-            </td>
-            <%= rank_td rank_average, "world" %>
-            <%= rank_td rank_average, "continent" %>
-            <%= rank_td rank_average, "country" %>
-            <td><%= odd_rank_reason if odd_rank_reason_needed?(rank_single, rank_average) %></td>
-          </tr>
+          <% if rank_single.event.rank < 990 %> 
```

Comment:
There's a dedicated [`official?`](https://github.com/thewca/worldcubeassociation.org/blob/3eb16bf8669ecb11ac20a89f27f75d83a7380f79/WcaOnRails/app/models/event.rb#L40-L42) method for that.

Reply:
Additionally, I think this filter should be done earlier: we're not doing anything with the `rank_single` if it's not official.
I'd either:
  - change [these lines](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/controllers/persons_controller.rb#L32-L33) in the controller to include something like `select { |r| r.event.official? }`
  - create some "sorted_official" scope in [here](https://github.com/thewca/worldcubeassociation.org/blob/2f2a1b591a64fcef5cfb5ad7e0ee01a1e61f4e21/WcaOnRails/app/models/ranks_single.rb#L3) that joins the event table and filter + sort the results. Which then could be used with `@person.rankSingle.sorted_official` I believe (and would remove the need for the `sort_by` above).

Reply:
Sure! Those sound good. Does the general idea of removing the exclusion in CAD sound like a good idea?

Reply:
From the top of my head I think so, hopefully we won't find any issues upon completing the next CAD after it's merged :)

Reply:
Oh btw I think the error from travis is the trailing whitespace on this specific line :)

Reply:
Iirc the test didn’t pass. I wanted to have the concept approved before fixing things up :D should get around to it this weekend!

Reply:
All Fixed up now :D, the tests that I originally thought was something else